modules/Hoot/ui/managePanel/jobs.js

```diff
@@ -503,46 +505,50 @@ export default class Jobs extends Tab {
                     }
                 });
 
-                if (d.jobType.toUpperCase() === 'DERIVE_CHANGESET') {
-                    //Get info for the derive
-                    actions.push({
-                        title: 'upload changeset',
-                        icon: 'cloud_upload',
-                        action: async () => {
-                            Hoot.api.differentialStats(d.jobId, false)
-                                .then( resp => {
-                                    this.diffStats = new DifferentialStats( d.jobId, resp.data ).render();
-
-                                    Hoot.events.once( 'modal-closed', () => delete this.diffStats );
-                                } )
-                                .catch( err => {
-                                    Hoot.message.alert( err );
-                                    return false;
-                                } );
-                        }
-                    });
-                }
 
-                if (d.jobType.toUpperCase() === 'CONFLATE') {
-                    let currentLayer = this.findLayer( d.mapId );
-
-                    if (currentLayer && currentLayer.grail) {
+                // Only advanced user may perform these
+                if (this.privileges && this.privileges.advanced === 'true') {
```

Comment:
check if actual user object received from services and put in local storage encodes advanced as a boolean

Reply:
I think I answered my own question.  Because we are using hstore in the db, everything key and value are stored as string.